notifications/rest/credentials/create-fcm-credential/create-fcm-credential.5.x.rb

```diff
@@ -0,0 +1,18 @@
+# Download the Ruby helper library from twilio.com/docs/libraries/ruby
+require 'twilio-ruby'
+
+# Get your Account Sid and Auth Token from https://www.twilio.com/console
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+
+# set up a client to talk to the Twilio REST API
+client = Twilio::REST::Client.new(account_sid, auth_token)
+
+# Create a credential for a given identity
+credential = client.notify.v1.credentials.create(
+  friendly_name: 'MyGCMCredential',
+  type: 'fcm',
+  # secret: 'fcm_secret'
```

Comment:
A question for you @acamino, in the curl request for this example there is a value name `Secret` which is sent to the server. However, I don't see this value anywhere in the _Twilio Ruby Helper Library_. Any idea how we could handle this?

https://github.com/TwilioDevEd/api-snippets/blob/master/notifications/rest/credentials/create-fcm-credential/create-fcm-credential.json.curl#L4

https://github.com/twilio/twilio-ruby/blob/5.0.0.rc17/lib/twilio-ruby/rest/notify/v1/credential.rb#L116-L124